stacksCollectible/index.ts

```diff
@@ -59,9 +60,10 @@ async function fetchBatchData(batch: Array<NonFungibleToken>, collectionRecord:
     const contractId = principal[0];
 
     if (contractInfo[1] === 'bns') {
-      //no further data required for BNS, arrange into collection
+      // no further data required for BNS, arrange into collection
+      // currently stacks only supports 1 bns name per address
```

Comment:
i know this was already merged but we shouldn't mutate a function param inside this fn [line 72](https://github.com/secretkeylabs/xverse-core/pull/274/files#diff-a02ab405a1cb56ad7eea17a73f1c8fb84e334bbcc8d5b332a44d21476c821a02R72) it can lead to undesired side effects imo

Reply:
yeah true